### DISCUSSION TITLE
AMD GPU

### DISCUSSION THREAD

**Discussion by rolandschulz** [intel/llvm#4194] [Q&A]
[Getting started](https://intel.github.io/llvm-docs/GetStartedGuide.html#build-dpc-toolchain-with-support-for-amd-rocm) states that only Vega20 gfx906 is being tested. Is this still the case? I have been able to find gfx1011 (Pro V520 on  AWS G4ad) and gfx900 (vega56, vegafe on gpueater or MI25  on genesiscloud) as cloud instances. Are either of those likely to work well? Or does someone know of any cloud instances with Vega20/gfx906?

@jchlanda @npmiller @AidanBeltonS do you know?

**Comment by bader** [comment]
Adding @malixian to this discussion.
I believe the statement to Getting Started Guide was originally added by @malixian.

**Comment by AerialMantis** [comment]
I believe the original patch to introduce the AMD GPU backend was tested on Vega20 (gfx906), and we have subsequently tested on Vega20 (gfx906) and Arcturus (gfx908). I am not aware of any other specific architectures being tested, so I can't confirm how well those would work, though it would be interesting to try them.
